### PR TITLE
cephfs-top: replace isinstance call with current_screen variable

### PR DESCRIPTION
**Pull request by joscollin**
It makes more sense to check the 'self.current_screen' variable as it's introduced to identify the `All Filesystem` and `Selected Filesystem` screens. We may rename this variable to `active_screen` or something if it's more appropriate. @neesingh-rh What do you think?

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins test make check

**Comment by joscollin** [comment]
jenkins test make check

**Review by neesingh-rh — COMMENTED** [review]
@joscollin I think this way is better as we do have `self.current_screen` for checks now. And I am good with your suggestion of changing it to `self.active_screen`. I know it won't crash the cephfs-top but will test the PR once with the changes.

**Comment by joscollin** [comment]
> @joscollin I think this way is better as we do have `self.current_screen` for checks now. And I am good with your suggestion of changing it to `self.active_screen`. I know it won't break the cephfs-top anywhere but will test the PR once with the changes.

Done the changes

File: src/tools/cephfs/top/cephfs-top [hunk 1]
```diff
@@ -943,10 +942,10 @@ class FSTop(object):
 
     def run_all_display(self):
         # clear text from the previous screen
-        if self.current_screen == FS_TOP_FS_SELECTED_APP:
+        if self.active_screen == FS_TOP_FS_SELECTED_APP:
```
Comment (neesingh-rh) [inline src/tools/cephfs/top/cephfs-top:945]: @joscollin What's the notion behind having a check before `self.header.erase()` on switching screens only in `run_all_display()` not in `self.run_display()`?

Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top:945]: It means the current run is a re-run from `run_display` and it needs a complete erase of the header. Please comment that code and check, you would see the previous text in the header.

Comment (neesingh-rh) [reply src/tools/cephfs/top/cephfs-top:945]: Yes, I understood. I want to ask why this check is not required in `run_display()`?

Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top:945]: For `run_display` the check is not required, as it's always erasing the header at the starting point. This erase is necessary for `run_display`. But an erase is not necessary for `run_all_display` when cephfs-top is launching. So basically the check saves an unnecessary erase call.

Comment (neesingh-rh) [reply src/tools/cephfs/top/cephfs-top:945]: Okay, we are saving an unnecessary call for erase when cephfs-top launches.

**Review by neesingh-rh — APPROVED** [review]

File: src/tools/cephfs/top/cephfs-top (hunk 1 above)
Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top:945]: yes

**Pull request merged**